packages/tasks/src/hardware.ts

```diff
@@ -72,6 +72,10 @@ export const SKUS = {
 				tflops: 12.0,
 				memory: [16],
 			},
+			"Quadro P6000": {
+				tflops: 20,
+				memory: [24],
+			},
```

Comment:
We are now aiming to sort in reverse chronological order (by family). Since this is a Pascal GPU (2016), can we move it below V100?

Reply:
Also, what's the source for the tflops number? I'm seeing lower float32 flops and much lower float16 speed here https://www.techpowerup.com/gpu-specs/quadro-p6000.c2865

Reply:
@pcuenca Yes move it to wherever, and I used techgage for the source on that card.